sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+
+We measure activity by activity within the GitHub repo. Filing/responding to issues and submitting/reviewing PRs.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+We currently have no metric for reviewer/approver bandwidth. We've worked in a communal manner to ensure nothing becomes stale. This has generally worked over the last several years, and isn't a major concern for us.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+The SIG doesn't participate in any activities or programs explicitly meant to help grow contributors. We attempt to be responsive and welcoming within our issues and PRs to try and retain those who take the first step and engage in conversation however.
+
+**What programs do you participate in for new contributors?**
+
+We attempt to maintain `Good First Issue` labels on things that we think early-lifecycle contributors could begin with. During KubeCons we also do introduction presentations and mentoring sessions to try and funnel contributors in to our SIG. 
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+Yes, current chairs are spread between Kubermatic (formerly Loodse) and Red Hat. 
+
+## Current initiatives and project health
+
+In addition to the ongoing effort of maintenance, we are improving support visualizing various resource objects and adding the ability for installations to have custom themes.
+
+The often-unspoken reality is, a UI or Dashboard for Kubernetes is often a differentiator for many companies. Additionally, there are many different ways and opinions on what a front-end for Kubernetes should look like.
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+When the Dashboard was originally written, it used a pull/polling design. Now, we are working towards supporting the shared informer pattern, making the Dashboard more-real-time.
+
+Another major area to highlight is we are continually onboarding new language translations for the Dashboard.
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+There have been no KEPs opened as part of SIG-UI.
+
+**What areas and/or subprojects does the group need the most help with?**
+
+Recruiting new contributors. Front-ends are very opinionated, and those with differing opinions seem to prefer starting from scratch rather than put in the effort in learning an existing project to contribute to. This has been observed time and time again.
```

Comment:
recruiting new contributors is going to require having meetings

Reply:
can you tell us about some of the skills folks would need in order to contribute so we can help with outreach? projects that we can specifically call out? the more clear we are with contributor asks, the better.

Reply:
Do you want me to add it in the report or as a comment here? (Tentatively I just put it within the doc)